sim/src/main/cc/firesim/firesim_top.cc

```diff
@@ -34,6 +36,9 @@ firesim_top_t::firesim_top_t(int argc, char** argv)
         if (arg.find("+zero-out-dram") == 0) {
             do_zero_out_dram = true;
         }
+        if (arg.find("+disable-asserts") == 0) {
```

Comment:
This should be handled in the assert bridge driver like plus args for other bridges. Remove the enable constructor parameter as well.